packages/web/hooks/ui-config/use-superfluid-pool.ts

```diff
@@ -17,7 +17,7 @@ export function useSuperfluidPool(): {
   const account = accountStore.getWallet(chainId);
 
   const delegateSharesToValidator = useCallback(
```

Comment:
There seems to be a typescript issue with React 18. `useCallback` should be able to infer the types from the type of the callback. IMO this is important to reduce clutter across the app. Generally, we should minimize type definitions and annotations as much as possible, if it's not possible to infer the type.